### ISSUE TITLE
Media Queries

### ISSUE THREAD

**Issue by griffinlacek** [mkn1889/f1-3-c2p1-colmar-academy#1]
You have media queries in your project for mobile styling and that is good! However, the queries fail sometimes due to the way they're setup.

Take for example this query:

https://github.com/mkn1889/f1-3-c2p1-colmar-academy/blob/master/ColmarAcademy/resources/css/style.css#L343

This query doesn't handle what to do after the width is 1250px or more. So, on my screen I have both your mobile header and desktop header because of this. There are two ways to combat this. Either, make sure that all mobile styles adhere to a strict max-width and display none all desktop styles (maybe something smaller like 640px) or design on the basis of mobile first. This is where your base styles are mobile styles and you specify min-width queries like

```
@media only screen and (min-width:640px){
  // desktop styles
```

Basically, by default your site is mobile ready and then you simply override those mobile styles with media queries for desktop sizes.

Something to consider!
